### DISCUSSION TITLE
更新日志 - 22.03.7 - 20241227

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#2010] [Announcements]
本次更新包含x86和rk35xx系列

# 设备：
* x86：修复上一版本luci界面列表选项没法清空保存的问题
* rk35xx：增加屏幕状态监控插件，HDMI接口接上显示器即可，暂无交互功能。如果需要禁用此功能，可以执行`uci set lcdsimple.@main[0].enabled=0; uci commit lcdsimple; /etc/init.d/lcdsimple stop`

**Comment by jinhzizhong** [comment]
升级了之后应用商店无法使用了

**Comment by b0070** [comment]
lcdsimple显示日志遮挡仪表盘，仪表盘同时也不读数据。

**Reply by jjm2473** [reply]
x86吗？目前没适配x86

**Comment by teochen** [comment]
升级新版后pppoe无法拨到公网 ip 了，降回 10月25 这个版本又可以播到公网 ip，大概是因为啥啊

**Comment by MR-WANGG** [comment]
大佬，群晖docker在哪来下载镜像包呀，或拉取地址？

**Comment by cppfuns** [comment]
升级后无法进入后台，拔电重启，恢复原来版本

**Comment by 275106590** [comment]
<img width="670" alt="1736944973793" src="https://github.com/user-attachments/assets/477a6c82-6afb-4b7d-8719-8afa941535ac" />
<img width="1661" alt="1736945027218" src="https://github.com/user-attachments/assets/75d07d7b-031e-4d95-8321-bfa93f84f3e0" />
友善R5S，接口这里显示了IPV6地址，主页不显示IPV6.用ipv6测试网站测试不通过，但是重启wan口后主页也显示了 测试网站也通过，但是几分钟后又不行

**Reply by orznz** [reply]
我这也是，很奇怪

**Reply by rootsysk** [reply]
你少了一个接口，wan6 DHCPv6 客户端

**Comment by vatebur** [comment]
感谢。终于能把HDMI接口和闲置显示器用上了。

**Comment by AaronChuzb** [comment]
屏幕可以适配x86吗

**Reply by b0070** [reply]
下一个版本会适配么？

**Comment by king6731253** [comment]
服务-硬盘休眠，功能提示错误：
`TypeError
null is not an object (evaluating 'disk[opt].trim')`
